user_docs/en/userGuide.t2t

```diff
@@ -1851,7 +1851,7 @@ The Add rule dialog also contains a checkbox to say whether or not you want the
 NVDA ignores case by default).
 
 Finally, a set of radio buttons allows you to tell NVDA whether your pattern should match anywhere, should only match if it is a complete word or should be treated as a "Regular expression".
-Setting the pattern to match as a whole word means that the replacement will only be made if the pattern does not occur as part of a larger word; i.e. a character other than an alphanumeric character or an underscore (or no character at all) comes both immediately before and after the pattern.
+Setting the pattern to match as a whole word means that the replacement will only be made if the pattern does not occur as part of a larger word. This condition is met if the characters immediately before and after the word are anything other than a letter, a number, or an underscore, or if there are no characters at all.

```

Comment:
To make translation of this file easier, we like each sentence to be on a new line. Could you split this line please?

Reply:
Ok!